test/tests.js

```diff
@@ -82,7 +82,14 @@ describe('Regex that includes emoji as their text representation', () => {
 	};
 
 	// Test a default text presentation character rendered as emoji.
+	// Hardcoded tests just to be safe, repeated by the scripted loop below.
 	test('\u{2194}');
 	test('\u{1F321}');
 
+	// Test `Emoji` symbols.
+	const Emoji = require('unicode-tr51/Emoji.js');
+	for (const codePoint of Emoji) {
+		test(String.fromCodePoint(codePoint));
+	}
+
```

Comment:
When I do that it produces 207 failures, as line 41 is in the section testing the `index` regex instead of the `text` one. It becomes a case of testing the `Emoji` symbols by themselves against the `index` regex that expects them to be always be followed by `\uFE0F`, hence the failures.